backend/config/default.cjs

```diff
@@ -191,6 +190,10 @@ module.exports = {
       text: '',
       startTimestamp: '',
     },
+
+    avScanning: {
```

Comment:
By default, we don't want avScanning enabled because you need to have an ClamAV instance running somewhere